woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1317,7 +1317,7 @@ def test_dataframe_methods_on_accessor_to_pandas(sample_df):
         pd_df = sample_df.ww.compute()
     elif _is_koalas_dataframe(sample_df):
         pd_df = sample_df.ww.to_pandas()
-
+        pd_df.ww.init(name='woodwork', index='id')
```

Comment:
I don't think we want to do this here. This is testing that if we call .compute or .to_panads through the Woodwork access that we get back a dataframe with Woodwork already initialized. If we aren't getting back an initialized dataframe with koalas, I think that is pointing at another issue we need to investigate.

Reply:
Yup! I brought this up in a standup last week and @tamargrey advised me to ignore it for now, and investigate it in another PR. Should I put up a new issue or fix it in this PR?

Reply:
I think it's ok to create a separate issue and PR for resolving this.

In the meantime, I think we would be better off to skip this test for Koalas with a message/comment explaining why we are skipping rather than reinitializing Woodwork since reinitializing basically means we aren't testing what this test was intended for.

Reply:
Created issue #1071!